index.html

```diff
@@ -72,6 +68,17 @@
     <!-- <article class="projects divs" id="project-2">&nbsp;
     </article> -->
 
+    <!-- JACKED TEMPLATE FROM 301d18 lab 2 -->
```

Comment:
You have a lot of commented out code. Best practice would be to not put this in your master. I would suggest if you haven't finished this part of the assignment, to keep the PR open until you do!